test/integration/full/landmark-no-more-than-one-main/frames/level1.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html id="pass2">
+<head>
+	<meta charset="utf8">
+	<script src="/axe.js"></script>
+</head>
+<body>
+	<p>No main content should still pass</p>
```

Comment:
Should this fail? I wouldn't think a `<main>` in an iframe would count towards the main document. But maybe @dylanb or @WilcoFiers have some input?